server/src/services/auth.service.ts

```diff
@@ -104,6 +104,13 @@ export class AuthService extends BaseService {
 
     const updatedUser = await this.userRepository.update(user.id, { password: hashedPassword });
 
+    if (dto.logOutOhterSessions) {
+      await this.eventRepository.emit('UserLogoutOtherSessions', {
+        userId: auth.user.id,
+        currentSessionId: auth.session?.id,
+      });
+    }
```

Comment:
And then you'd always emit the change password event (could be helpful in the future for other stuff too)